docs/usage/config-overview.md

```diff
@@ -0,0 +1,250 @@
+- You onboarded many repos prior to having a shared org config, and don't want to retrospectively edit each repo's config
+
+#### How it's found
+
+If `inheritConfig` is `true` in Global config then Renovate will look for Inherited config before processing each repository.
+The repository and file name which Renovate looks for can be configured using the other `inheritConfig*` settings documented in Global config.
+Default values are `{{parentOrg}}/renovate-config` for repository name and `org-inherited-config.json` for file name.
+
+If found, Inherited config will be merged on top (i.e. override) Global config, however note that there are many settings which are global-only so attempting to configure one of those will result in an error.
+
+Inherited config can include all Repository config settings as well as any Global config options which are documented as "supportsInheritConfig".
+
+For information on how the Mend Renovate App supports Inherited config, see the dedicated "Mend Renovate App Config" section toward the end of this page.
+
+### Repository config
+
+Repository config is the config loaded from the repository itself.
+Alternative file names are supported, but the default is `renovate.json`.
+If multiple such file names are found in the same repository then Renovate will use the first one it finds and ignore the other(s).
+
+### Config precedence
+
+Once Repository config is loaded, it is merged over the top of the previously loaded Global and Inherited config, meaning it takes precedence over them.
+Presets referenced with an "extends" config are resolved first and take lower precedence over regular/raw config in the same file or config object.
+
+## Onboarding
+
+When Renovate processes a repository, one of the first decisions it makes is "Does this repository need to be onboarded?".
+By default, Renovate will generate an "Onboarding PR" with a default config if a repository does not already have a Repository config file committed to the default branch.
+
+### Onboarding Config
+
+When Renovate creates an Onboarding PR it will propose a Repository config file to be merged.
+By default, it is essentially an empty config with only the Renovate JSON schema referenced, but you can change this behavior if desired.
+
+If you configure `onboardingConfig` in either Global config or Inherited config then Renovate will use that config directly instead of the default.
+
+Alternatively if follow Renovate's naming convention for shared presets then it can automatically detect those instead.
+If the repository `{{parentOrg}}/renovate-config` contains a `default.json` file then this will be treated as the organization's default preset and included in the Onboarding config.
+Additionally for platforms which support nested Organization/Group hierarchies, Renovate will "hunt" up such hierarchies for a `renovate-config` repository with default config and stop when it finds the first.
+
+<!-- prettier-ignore -->
+!!! note
```

Comment:
```suggestion
!!! warn
```

I think we should warn here? As we also warn in logs?

Reply:
I think note is enough. We have already warned about this in other places